values.yaml

```diff
@@ -351,6 +351,20 @@ svc_sema:
 #            average: 10
 #            burst: 20
 
+# ----------------------------------------------------------------------
+# PROJECT SERVICE CONFIGURATION
+# ----------------------------------------------------------------------
+svc_project:
+  replicas: 4
```

Comment:
Are we setting the default replicas all to 1?

Reply:
changed